package.json

```diff
@@ -6,7 +6,7 @@
     "deduplicate": "node scripts/deduplicate.js",
     "argos": "argos upload test/regressions/screenshots/chrome --token $ARGOS_TOKEN",
     "build:codesandbox": "lerna run --parallel --scope \"@material-ui/*\" build",
-    "docs:api": "rimraf ./docs/pages/api && cross-env BABEL_ENV=test __NEXT_EXPORT_TRAILING_SLASH=true babel-node ./docs/scripts/buildApi.js ./packages/material-ui/src ./docs/pages/api-docs && cross-env BABEL_ENV=test __NEXT_EXPORT_TRAILING_SLASH=true babel-node ./docs/scripts/buildApi.js ./packages/material-ui-lab/src ./docs/pages/api-docs",
+    "docs:api": "rimraf ./docs/pages/api && cross-env BABEL_ENV=test __NEXT_EXPORT_TRAILING_SLASH=true babel-node ./docs/scripts/buildApi.js ./packages/material-ui/src ./docs/pages/api-docs && cross-env BABEL_ENV=test __NEXT_EXPORT_TRAILING_SLASH=true babel-node ./docs/scripts/buildApi.js ./packages/material-ui-lab/src ./docs/pages/api-docs && yarn prettier",
```

Comment:
It's formatting the code that is generated (.d.ts).

Reply:
It seems that we have the capability to output a correct format, right from the `docs/scripts/buildApi.js` file. What do you think of going down this road rather than requiring to run prettier to remove the trailing spaces? Not really important.

Reply:
Could you describe what issue you have with this approach? I don't have the same context as you.